heeji/LV_1/행렬의-덧셈.js

```diff
@@ -0,0 +1,11 @@
+function solution(arr1, arr2) {
+    let answer = []; 
+    for(let i = 0; i < arr1.length; i++) {
+        let newArray = [];
+        for (let j = 0; j < arr1[i].length; j++) {
+            newArray.push(arr1[i][j] + arr2[i][j]);
+        }
+        answer.push(newArray);
+    }
+    return answer;
+}
```

Comment:
```suggestion
function solution(arr1, arr2) {
    const answer = [[]];
    for(let i = 0; i < arr1.length; i ++) {
        answer[i] = []
        for(let j = 0; j < arr1[i].length; j++) {
            answer[i].push(arr1[i][j] + arr2[i][j])
        }
    }
    return answer;
}
```
이렇게 answer 안에 빈 배열을 하나 둠으로써 변수를 하나만 선언하고 해결할 수 있습니다! for문을 활용해서 잘 푸셨네요👍

Reply:
answer안에 빈 배열을 두고 어떻게 사용해야하나 했는데 피드백 감사합니닷!